redeem/gcodes/M2x.py

```diff
@@ -118,10 +118,12 @@ def execute(self, g):
             return
 
         # list all files on the device
-        # self.printer.send_message(g.prot, "files on external memory '{}'".format(list_location))
+        self.printer.send_message(g.prot, "Begin file list:")
         for root, directories, filenames in os.walk(list_location):
             for filename in filenames:
-                self.printer.send_message(g.prot, "{}/{} -".format(list_location, filename))
+		file_line_count = sum(1 for line in open(list_location + "/" + filename))
```

Comment:
There seem to be mixed tabs and spaces here.

Reply:
Oops, likely. I'll fix those everywhere in the file.